generators/eslint/index.js

```diff
@@ -30,12 +30,12 @@ module.exports = generators.Base.extend({
       }
     };
     var devDep = {
```

Comment:
Can we add eslint to the devDep - I think it's a peerDeps. Also, having it here allows editor linters to work (at least with Atom and Sublime text)